recipe/build.sh

```diff
@@ -5,8 +5,8 @@ mkdir $PREFIX/lib
 sh ./configure --prefix=$PREFIX \
     --without-debug --without-ada --without-manpages \
     --with-shared --disable-overwrite --enable-termcap \
-    --with-termlib
-
+    --with-termlib \
+    --enable-widec --with-terminfo-dirs=/usr/share/terminfo
```

Comment:
Turns out specifying terminfo-dirs was a bad idea. As a result, we have to drop it. See PR ( https://github.com/conda-forge/ncurses-feedstock/pull/19 ) for more details and xref'd issue.